rundeckapp/grails-app/views/menu/home.gsp

```diff
@@ -462,9 +459,7 @@
 	</div>
 	<!-- VUE JS MODULES -->
 	<asset:stylesheet href="static/css/pages/home.css"/>
-	<asset:stylesheet href="static/css/components/first-run.css"/>
```

Comment:
This file isn't being created, but the import isn't needed as the CSS required by its components is included in the chunk-common.css